spicy/CMakeLists.txt

```diff
@@ -193,6 +193,23 @@ if ( HILTI_USE_JIT )
     target_link_libraries(spicy-driver PRIVATE spicy)
 endif ()
 
+set(SPICY_EXEUTABLES "spicyc" "spicy-config" "spicy-driver" "spicy-doc")
```

Comment:
This misses a number of executable (e.g., `spicyz`), but like I wrote before, manually exposing these symbols via general linker flags would be preferrable.